features/admin.connections.v1/components/edit/forms/authenticators/saml-authenticator-form-finalform.tsx

```diff
@@ -0,0 +1,1108 @@
+            NameIDType: findPropVal<string>({
+                defaultValue: findMeta({ key: "NameIDType" })?.defaultValue ?? DEFAULT_NAME_ID_FORMAT,
+                key: "NameIDType"
+            }),
+            RequestMethod: findPropVal<string>({
+                defaultValue: findMeta({ key: "RequestMethod" })?.defaultValue ?? DEFAULT_PROTOCOL_BINDING,
+                key: "RequestMethod"
+            }),
+            ResponseAuthnContextClassRef: findPropVal<string>({
+                defaultValue: "string",
+                key: "ResponseAuthnContextClassRef"
+            }),
+            SPEntityId: findPropVal<string>({ defaultValue: "", key: "SPEntityId" }),
+            SSOUrl: findPropVal<string>({ defaultValue: "", key: "SSOUrl" }),
+            SignatureAlgorithm: findPropVal<string>({ defaultValue: "RSA with SHA256", key: "SignatureAlgorithm" }),
+            commonAuthQueryParams: findPropVal<string>({ defaultValue: "", key: "commonAuthQueryParams" }),
+            /**
+             * https://github.com/wso2/product-is/issues/17004
+             */
+            isAssertionSigned: findPropVal<boolean>({ defaultValue: false, key: "isAssertionSigned" })
+        } as SamlPropertiesInterface;
+    }, []);
+
+    const disabledFeatures: string[] = useSelector(
+        (state: AppState) => state.config.ui.features?.identityProviders?.disabledFeatures
+    );
+
+    const getAlgorithmsDropdownFieldValidators = (): ((value: string) => string | undefined) => {
+        // if (isLogoutReqSigned || isAuthnReqSigned) {
+        //     return required;
+        // }
+
+        return (/* No validations */) => void 0;
+    };
+
+    const getIncludeAuthenticationContextOptions = (): {
+        label: string;
+        value: string;
+    }[] => {
+        return [
+            { label: "Yes", value: "yes" },
+            { label: "No", value: "no" },
+            { label: "As request", value: "as_request" }
+        ];
+    };
+
+    const getForceAuthenticationOptions = (): {
+        label: string;
+        value: string;
+    }[] => {
+        return [
+            { label: "Yes", value: "yes" },
+            { label: "No", value: "no" },
+            { label: "As request", value: "no-authn" }
+        ];
+    };
+
+    const getResponseAuthnContextClassRefOptions = (): {
+        label: string;
+        value: string;
+    }[] => {
+        return [
+            { label: "Default", value: "default" },
+            { label: "As response", value: "as_response" }
+        ];
+    };
+
+    const getAvailableAuthContextComparisonLevelOptions = (): DropdownChild[] => {
+        return [
+            { key: 1, text: "Exact", value: "Exact" },
+            { key: 2, text: "Minimum", value: "Minimum" },
+            { key: 3, text: "Maximum", value: "Maximum" },
+            { key: 4, text: "Better", value: "Better" }
+        ];
+    };
+
+    const authenticationContextClassOptions: DropdownChild[] = [
```

Comment:
Need to check whether there is a way to get the available authentication context class options from a backend API.

cc: @NipuniBhagya